app/src/pages/inside/logsPage/logToolbar/logToolbar.scss

```diff
@@ -35,3 +35,13 @@
 .left-arrow-button {
   margin-right: -1px;
 }
+
+.breadcrumbs-wrapper {
```

Comment:
After providing custom class to the `InputCheckbox` component it will be unnecessary.